constants/tasks.js

```diff
@@ -5,11 +5,23 @@ const TASK_TYPE = {
 }
 
 const TASK_STATUS = {
-  ACTIVE: 'active',
-  ASSIGNED: 'assigned',
-  BLOCKED: 'blocked',
-  COMPLETED: 'completed',
-  UNASSIGNED: 'unAssigned'
+  ACTIVE: 'IN_PROGRESS',
+  IN_PROGRESS: 'IN_PROGRESS',
+  PENDING: 'PENDING',
+  ASSIGNED: 'ASSIGNED',
+  BLOCKED: 'BLOCKED',
+  COMPLETED: 'COMPLETED',
+  UNASSIGNED: 'AVAILABLE',
+  AVAILABLE: 'AVAILABLE',
+  SMOKE_TESTING: 'SMOKE_TESTING',
+  NEEDS_REVIEW: 'NEEDS_REVIEW',
+  IN_REVIEW: 'IN_REVIEW',
+  APPROVED: 'APPROVED',
+  MERGED: 'MERGED',
+  SANITY_CHECK: 'SANITY_CHECK',
+  REGRESSION_CHECK: 'REGRESSION_CHECK',
+  RELEASED: 'RELEASED',
+  VERIFIED: 'VERIFIED'
 }
```

Comment:
What can we do for the tasks that are currently set as **High Priority**  also the tasks that might not fit in this list in that case status will be undefined

Reply:
I am handling the `high priority` status in new commit.

Reply:
I checked the current /task api and it has only 4 status values which we are handling already.